runtime/databricks/automl_runtime/forecast/pmdarima/model.py

```diff
@@ -281,7 +318,7 @@ def predict(self, context: mlflow.pyfunc.model.PythonModelContext, model_input:
         """
         self._validate_cols(model_input, self._id_cols + [self._time_col])
         df = model_input.copy()
-        df["ts_id"] = df[self._id_cols].apply(lambda r: "-".join(r.values.astype(str)), axis=1)
+        df["ts_id"] = df[self._id_cols].apply(tuple, axis=1)
```

Comment:
Nit: Is it possible to use a UUID based or at least a constant for this "ts-id", since this may have trouble if user data frame has column called "ts_id". And use hard code string can sometimes write bugs in future development.

Reply:
This is also a issue from prophet which reserved multiple column names. We can change this as a followup.